components/centos/centos-docker-base-setup/Vagrantfile

```diff
@@ -53,4 +53,5 @@ Vagrant.configure(2) do |config|
     config.ssh.private_key_path = ENV['OS_PRIVATE_KEYPATH']
     config.ssh.username = 'vagrant'
   end
+  config.servicemanager.services = 'docker'
```

Comment:
VSM configures only `docker` service by default for ADB box, we do not need to explicitly mention it, although we can comment out these lines in Vagrantfile to give hints to users.